src/liapunovspectrum.jl

```diff
@@ -80,12 +70,11 @@ function classicalGS(A)
     return Q, R
 end
 # Modified Gram–Schmidt (Trefethen algorithm 8.1)
-function modifiedGS(A)
+function modifiedGS!(A, Q, R, aⱼ, qᵢ)
     m,n = size(A)
-    Q = similar(A)
-    R = zeros(eltype(A),n,n)
-    aⱼ = Array{eltype(A)}(m)
-    qᵢ = similar(aⱼ)
+    R[:,:] = zeros(eltype(A),n,n)
```

Comment:
Before commiting this change I made some tests with `R = ...` instead of `R[:,:] = ...` but it wasn't working so that's why I put the `[:,:]`; using `view` sounds nice, I'll try it!